docs/web/frontend/README.md

```diff
@@ -38,7 +38,8 @@ It is also important to keep in mind, especially as the ecosystem of front end d
 - \[ \] [DayJS](https://day.js.org/), [luxon](https://moment.github.io/luxon/index.html) (newer alternatives to Moment, working with dates and times)
   - Note: [MomentJS](https://momentjs.com/) has been deprecated, **do not use on new projects**
 - \[ \] [validateJS](https://validatejs.org/), [yup](https://github.com/jquense/yup) (validating form object models)
-- \[ \] [Formik](https://formik.org/), [React-hook-form](https://react-hook-form.com/) (building forms in React)
```

Comment:
Thought: I wonder if we should remove Formik from this line. What do you think?

Reply:
@gidjin Maybe I'm misunderstanding, but that's what is happening on that line